tox.ini

```diff
@@ -41,8 +41,8 @@ commands = black {posargs} qiskit_dynamics test
 usedevelop = False
 deps =
     -r{toxinidir}/requirements-dev.txt
-    jax<=0.4.6
-    jaxlib<=0.4.6
+    jax<=0.4.3
```

Comment:
You might want a comment here to explain the lower pin. Or maybe you don't need the lower pin if you follow my suggestion of checking the environment variable before warning.

Reply:
I'll implement your suggestion, making this moot.